soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -311,29 +311,75 @@ struct uart_reg {
  * Multi-Input Wake-Up Unit (MIWU) device registers
  */
 
-/* MIWU multi-registers */
-#define NPCX_WKEDG_OFFSET(n)    (0x000 + ((n) * 2L) + ((n) < 5 ? 0 : 0x1E))
-#define NPCX_WKAEDG_OFFSET(n)   (0x001 + ((n) * 2L) + ((n) < 5 ? 0 : 0x1E))
-#define NPCX_WKPND_OFFSET(n)    (0x00A + ((n) * 4L) + ((n) < 5 ? 0 : 0x10))
-#define NPCX_WKPCL_OFFSET(n)    (0x00C + ((n) * 4L) + ((n) < 5 ? 0 : 0x10))
-#define NPCX_WKEN_OFFSET(n)     (0x01E + ((n) * 2L) + ((n) < 5 ? 0 : 0x12))
-#define NPCX_WKINEN_OFFSET(n)   (0x01F + ((n) * 2L) + ((n) < 5 ? 0 : 0x12))
-#define NPCX_WKMOD_OFFSET(n)    (0x070 + (n))
+/* MIWU internal inline functions for multi-registers */
+static inline uint32_t npcx_wkedg_offset(uint32_t group)
+{
+	if (IS_ENABLED(CONFIG_SOC_SERIES_NPCX7)) {
+		return (0x000 + (group * 2ul) + (group < 5 ? 0 : 0x1e));
```

Comment:
shoulder drop the outer () on all of these.

'return' is not a function.

Reply:
Done.